spec/support/custom_data_save_period.rb

```diff
@@ -0,0 +1,9 @@
+module Stormpath
```

Comment:
I'd love to see a comment that explains what is going on here, future developers will be grateful.